Makefile

```diff
@@ -21,6 +21,13 @@ clear:
 	rm -f bin/*
 	rm -rf _out
 
+.PHONY: install
+install: static-release
+	sudo install -D -m 755 bin/containerd-nydus-grpc /usr/local/bin/containerd-nydus-grpc
+	sudo mkdir -p /etc/nydus
+	sudo cp misc/snapshotter/nydusd-config.json /etc/nydus/config.json
+	sudo systemctl enable ${PWD}/misc/snapshotter/nydus-snapshotter.service
```

Comment:
Do we want to install this systemd service file to system? e.g. /lib/systemd/system/nydus-snapshotter.service ?

Reply:
Sorry for previously misunderstanding. It is now installed.